### PR TITLE
fix: show correct error message when `--guest-space-id` option is missing

### PR DESCRIPTION
**Pull request by ryounasso**
## Why

When we try to export records from the App in the guest space, the error will be the following.
```
cli-kintone record export --base-url https://<subdomain>.cybozu.com --app 399 --api-token <token>
ERROR: KintoneRestAPIError: [520] [GAIA_IL23] ゲストスペース内のアプリを操作する場合は、リクエストの送信先を「/k/guest/（ゲストスペースのID）/v1/...」にします。 (WatJ7WqRh0mlkbEw6QdR)
```
We think this error message is incorrect. I want to use `spcify --guest-space-id`.
That's why I want to improve this error message.

## What

- I created new class `RunError` to wrap other errors with `CliKintoneError`.
- The process of converting errors from external packages to strings is summarized in the `CliKintoneError` class.
- Delete classes that are not longer needed.

## How to test

- Created tests of the changes.

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: src/record/export/index.ts [hunk 1]
```diff
@@ -60,7 +61,8 @@ export const run: (
       attachmentsDir,
     });
   } catch (e) {
-    logger.error(e);
+    // logger.error(e);
```
Comment (ueokande) [inline src/record/export/index.ts:64]:
```suggestion
```

File: src/utils/error.ts [hunk 2]
```diff
@@ -17,12 +22,39 @@ export abstract class CliKintoneError extends Error {
...
       return this.cause.toString();
+    } else if (this.cause instanceof KintoneAllRecordsError) {
+      return this._toStringKintoneAllRecordsError(this.cause);
+    } else if (this.cause instanceof KintoneRestAPIError) {
+      return this._toStringKintoneRestAPIError(this.cause);
     }
     return this.cause + "\n";
   }
 
+  protected _toStringKintoneAllRecordsError(
```
Comment (ueokande) [inline src/utils/error.ts:33]: Make it private
```suggestion
  private _toStringKintoneAllRecordsError(
```

**Review by ueokande — CHANGES_REQUESTED** [review]

File: src/record/export/index.ts (hunk 1 above)
Comment (ryounasso) [reply src/record/export/index.ts]: Fixed with a557a9d367ac6093128396a4c381b0ca5a23a5bd

File: src/utils/error.ts (hunk 2 above)
Comment (ryounasso) [reply src/utils/error.ts]: Fixed with a557a9d367ac6093128396a4c381b0ca5a23a5bd

**Review by tasshi-me — CHANGES_REQUESTED** [review]

File: src/record/import/usecases/__tests__/add/error.test.ts [hunk 3]
```diff
@@ -30,10 +30,23 @@ const schema: RecordSchema = {
...
+    // 60件のレコードをインポートします = numOfAllRecords
+    // cli-kintoneのaddRecords()でn件ずつrest-api-clientに送信
+    // rest-api-clientはレコードをm件ずつkintoneに送信
+    // 全体の44件目でエラーが発生した = errorIndex CSV上の46行目
+    // 先頭の10件はすでにインポート済み = numOfAlreadyImportedRecords
+    // 処理中のn件のうち、30件はインポート済み = numOfProcessedRecords
+    const numOfAllRecords = 60; // インポート処理対象のレコード全件 60件 CSV上の1行目-61行目
+    const numOfAlreadyImportedRecords = 10; // cli-kintone内で処理に成功したレコードの件数 10件
+    const numOfProcessedRecords = 30; // 現在処理中のチャンクのうち、処理に成功したレコードの件数 30件
+    const errorIndex = 44; // インポート対象のレコード全件のうち、エラーが発生したレコードのインデックス
```
[the remark this answers is not in the document]
Comment (ryounasso) [reply src/record/import/usecases/__tests__/add/error.test.ts]: Fixed with 2785a5481e577ce92ecd6099ade72601f7957606

File: src/record/export/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryounasso) [reply src/record/export/index.ts]: Fixed with a557a9d367ac6093128396a4c381b0ca5a23a5bd

**Review by ueokande — APPROVED** [review]

**Review by tasshi-me — APPROVED** [review]
LGTM 👍

**Pull request merged**